pkg/profiler/cpu/bpf/maps/maps.go

```diff
@@ -1821,6 +1823,44 @@ func (m *Maps) allocateNewShard() error {
 	return nil
 }
 
+// Get the largest chunk of `ut` (up to `maxLen`) that
```

Comment:
FYI some linters require the first word of a function doc comment to be the function name itself, apparently we don't have such a linter which surprises me.